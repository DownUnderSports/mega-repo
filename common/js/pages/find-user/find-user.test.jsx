import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { MemoryRouter as Router } from 'react-router-dom';

import FindUser from './find-user';

describe('Pages - FindUser', () => {
  const div = document.createElement('div');
  const createInfokit = (params = {}) => {
    ReactDOM.render(
      <Router>
        <FindUser
          match={{
            params: {
              ...params
            }
          }}
        />
      </Router>
    , div);
    return div.querySelector('p')
  }

  it('is snapshotable', () => {
    const tree = renderer
      .create(
        <Router>
          <FindUser />
        </Router>
      )
      .toJSON();
    expect(tree).toMatchSnapshot()
  })
})
