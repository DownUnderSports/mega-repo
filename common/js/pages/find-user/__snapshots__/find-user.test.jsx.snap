// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages - FindUser is snapshotable 1`] = `
<div
  className="infokit-page"
>
  <div
    className="row form-group"
  >
    <div
      className="col"
    >
      <div
        style={
          Object {
            "display": "block",
            "width": "100%",
          }
        }
      >
        <h1
          className="text-center"
        >
          SUBMITTING...
        </h1>
        <div
          className="la-ball-atom la-vw-sm"
          style={
            Object {
              "color": "#00F",
              "marginLeft": "auto",
              "marginRight": "auto",
              "zIndex": 0,
            }
          }
        >
          <div />
          <div />
          <div />
          <div />
        </div>
      </div>
    </div>
  </div>
</div>
`;
